docs/protocol/ibc.md

```diff
@@ -0,0 +1,21 @@
+<!--
+order: 5
+-->
+
+# Current IBC channels between Evmos and other chains
+
+IBC channels are added based on community consensus.
```

Comment:
can you add the link to the relayers page on Mintscan?

Reply:
done